client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/useInvestigatedPersonInfo.ts

```diff
@@ -46,8 +46,21 @@ const useInvestigatedPersonInfo = (): InvestigatedPersonInfoOutcome => {
         timeout(1900).then(()=> history.push(landingPageRoute));
     };
 
+    const getPersonAge = (birthDate: Date) => {
+
+        const currentDate = new Date();
+        let personAge = currentDate.getFullYear() - birthDate.getFullYear();
+        const monthDelta = currentDate.getMonth() - birthDate.getMonth();
+        if (monthDelta < 0 || (monthDelta === 0 && currentDate.getDate() < birthDate.getDate())) 
+        {
+            personAge--;
+        }
+        return String(personAge);
+    }
```

Comment:
var birthdate = new Date("1990/1/1");
var cur = new Date();
var diff = cur-birthdate; // This is the difference in milliseconds
var age = Math.floor(diff/31557600000); // Divide by 1000*60*60*24*365.25

לדעתי זה דרך יותר יפה אם היא עובדת אבל די לבחירתך

Reply:
מה מה מה? באמת